### ISSUE TITLE
New inspection level - SKIP

### ISSUE THREAD

**Issue by mccartney** [scapegoat-scala/scapegoat#646]
Current [inspection levels](https://github.com/scapegoat-scala/scapegoat/blob/master/src/main/scala/com/sksamuel/scapegoat/Level.scala) are: `ERROR`, `WARNING`, `INFO`. 

I suggest to add a new one: (working name) `SKIP`. So that for instance one can set it as a default level for some inspections. I think that has the potential of simplifying the configuration - i.e. instead of disabling inspections, one would set their level to `SKIP`.

**Comment by eugene-sy** [comment]
It feels like duplication to me. 
The use case when one wants to see the issue in the output but not break the build is covered by `INFO`.
If one would not like to see the issue in the report can be achieved by disabling the inspection.
Do I miss a scenario?

**Comment by mccartney** [comment]
You are right. My point is that the notion of disabling the inspection could be replaced with a new level.

This way we wouldn't need to develop and maintain two separate sets of configuration properties which overlap. E.g. we currently have default values for levels, but no support for a inspection to be disabled by default, etc.
